v2/holiday.go

```diff
@@ -117,6 +119,9 @@ func (h *Holiday) Calc(year int) (actual, observed time.Time) {
 		}
 	}
 	actual = h.Func(h, year)
+	if h.OffsetDays != 0 {
+		actual = time.Date(actual.Year(), actual.Month(), actual.Day()+h.OffsetDays, 0, 0, 0, 0, DefaultLoc)
```

Comment:
You can use `time.AddDate(0, 0, h.OffsetDays)` to simplify this.

Reply:
Changed to use `AddDate`